src/components/ui/BlueBar.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div class="blue-bar__card">
```

Comment:
background color can be passed as props (class or just as v-bind:style=...)

Reply:
done